src/utils/daos_autotest.c

```diff
@@ -98,18 +98,19 @@ step_new(int step, char *msg)
 {
 	int i;
 
-	printf("%3d  %s", step, msg);
+	fprintf(autotest_ap->outstream, "%3d  %s", step, msg);
 	for (i = strlen(msg); i < 25; i++)
-		printf(" ");
+		fprintf(autotest_ap->outstream, " ");
 	fflush(stdout);
```

Comment:
What are you fflushing here?

Reply:
That was left in by mistake, will remove it.